test/cron/daily-message.js

```diff
@@ -33,47 +38,70 @@ describe('daily-message cron job', function () {
   });
 
   it('should send a message the Friday before release week', async function() {
-    await assertMessageForDate('2020-07-13', '2020-07-20', 'Release week starts next week on 2020-07-20 are we all prepared? :lts:');
+    await assertMessageForDate({
+      todayDate: '2020-07-13',
+      releaseDate: '2020-07-20',
+      message:
+        'Release week starts next week on 2020-07-20 are we all prepared? :lts:',
+    });
   });
 
   it('should send a message on the first day of release week', async function () {
-    await assertMessageForDate('2020-07-20', '2020-07-20', `:tada: It's release week!! :tada: To see who is done and who isn't you can say '!release done' and I'll tell you who still needs to do something!
-
-If you know a team is done you can say '!release done <team>' where team can be blog, framework, data, or cli`);
+    await assertMessageForDate({
+      todayDate: '2020-07-20',
+      releaseDate: '2020-07-20',
+      message: `:tada: It's release week!! :tada: To see who is done and who isn't you can say '!release done' and I'll tell you who still needs to do something! If you know a team is done you can say '!release done <team>' where team can be blog, framework, data, or cli`,
```

Comment:
I think the newline in line 41 (deleted) might have been deliberately added. To keep the behavior the same, but to format the code better, what do you think about this approach?

```suggestion
      message: [
        `:tada: It's release week!! :tada: To see who is done and who isn't you can say '!release done' and I'll tell you who still needs to do something!`,
        '',
        `If you know a team is done you can say '!release done <team>' where team can be blog, framework, data, or cli`
      ].join('\n'),
```

Reply:
Ah I see. I should have clarified that. I like your approach with `.join()`. I'll update now.